core/src/test/java/com/linecorp/armeria/server/GlobPathMappingTest.java

```diff
@@ -75,13 +74,6 @@ void testRelativePattern() {
         mustFail("bar/baz", "/bar/baz/", "/foo/bar/baz/", "/foo/bar/baz/quo");
     }
 
-    @Test
-    void testPathValidation() {
-        final Route route = glob("**");
-        assertThatThrownBy(() -> route.apply(create("not/an/absolute/path"), false))
-                .isInstanceOf(IllegalArgumentException.class);
-    }
-
```

Comment:
This test has been removed because it's not possible to create a `RoutingContext` with an invalid path anymore. `DefaultRoutingContext.of()` returns a `RequestTarget`.